Marlin/src/feature/fwretract.cpp

```diff
@@ -53,8 +53,9 @@ float FWRetract::retract_length,                     // M207 S - G10 Retract len
       FWRetract::retract_recover_feedrate_mm_s,      // M208 F - G11 Recover feedrate
       FWRetract::swap_retract_length,                // M207 W - G10 Swap Retract length
       FWRetract::swap_retract_recover_length,        // M208 W - G11 Swap Recover length
-      FWRetract::swap_retract_recover_feedrate_mm_s, // M208 R - G11 Swap Recover feedrate
-      FWRetract::hop_amount;
+      FWRetract::swap_retract_recover_feedrate_mm_s; // M208 R - G11 Swap Recover feedrate
+float FWRetract::current_retract[EXTRUDERS],         // Retract value used by planner
```

Comment:
We only need one `current_retract` value, since only one extruder may be active at once, and we don't need to support situations like **retract ... T1 ... ... T0 ... recover**.